phys2bids/io.py

```diff
@@ -292,13 +214,67 @@ def read_header_and_channels(filename):
     return header, channel_list
 
 
-def load_txt_ext(filename, chtrig=0):
+def extract_header_items(channel_list, header=[]):
+    """
+    Process items header and channel_list depending on the format (acqKnowledge and labchart).
+
+    Parameters
+    ----------
+    channel_list: list
+        list with channels. For one of the formats it contains header information
+    header: list
+        list with that contains file header
+
+    Returns
+    -------
+    interval: list
+        contains interval or frequency value and units
```

Comment:
This description confuses me a bit. It has the interval _or_ frequency value _and_ units. So is it a list of dictionaries? Or tuples?

I also don't think there's any need to say "contains". I'd say something like "interval or frequency value for each channel."

Reply:
```
    interval : list of strings
        maximun sampling frequency or interval value and unit for the recording
```
Better? it only contains the value and the unit of one frequency or interval